javascript/react_js/react_testing_part_two.md

```diff
@@ -119,6 +119,14 @@ By just going through the code, it should give us some idea of what to test. It
 
 Go through its test file, [submissions-list.test.jsx](https://github.com/TheOdinProject/theodinproject/blob/main/app/javascript/components/project-submissions/components/__tests__/submissions-list.test.jsx). Again, don't worry if all of it doesn't make sense, we'll chew over it shortly.
 
+<div class="lesson-note" markdown="1">
+
+#### Note
+
+While the test suite above uses `data-test-id` to identify mocked child components, it must be remembered that the React Testing Library instead uses `data-testid` by default. This spelling was overridden for development convenience in a configuration file used to set up the test environment but this is not necessary.
```

Comment:
Thanks for doing this, just one suggestion to simplify it, I don't think we need the second sentence.

```suggestion
While the test suite above uses `data-test-id` to identify mocked child components, it must be remembered that the React Testing Library instead uses `data-testid` by default.
```